### DISCUSSION TITLE
使用Dockerfile构建镜像 | hengshuai's blog

### DISCUSSION THREAD

**Discussion by giscus** [ihengshuai/blog#91] [Announcements]
# 使用Dockerfile构建镜像 | hengshuai's blog

Dockerfile是一个文本文件里面包含了一条条构建镜像的指令，使用Dockerfile构建镜像更加方便、间接、透明

https://blog.usword.cn/fullstack/docker/dockerfile.html

**Comment by ihengshuai** [comment]
若对Dockerfile有一些疑问，可以留言
